func_importvalue.go

```diff
@@ -0,0 +1,43 @@
+package cloudformation
+
+// ImportValue returns a new instance of ImportValue that imports valueToImport.
+func ImportValue(valueToImport string) ImportValueFunc {
```

Comment:
I'm not sure - I can't tell if if's the `Ref` or the `Ref` param that potentially causes a problem:
```
Similarly, the ImportValue function can't include functions (Ref or GetAtt) that depend on a resource.
```